apps/balancer-tools/src/app/poolsimulator/(components)/Curves/ImpactCurve.tsx

```diff
@@ -186,6 +190,27 @@ export function ImpactCurve() {
     };
   };
 
+  const createBetaLimitsDataObject = (
+    x: number[],
+    y: number[],
+    legendGroup: string,
+  ) => {
+    // https://docs.xave.co/product-overview-1/fxpools/amm-faqs
```

Comment:
this link should be where in the place where we calculate these points instead of here right?